hi-ml/src/health_ml/runner.py

```diff
@@ -216,6 +216,7 @@ def after_submission_hook(azure_run: Run) -> None:
 
         # TODO: Update environment variables
         environment_variables: Dict[str, Any] = {}
+        environment_variables[DEBUG_DDP_ENV_VAR] = self.experiment_config.debug_ddp.value  # type: ignore
```

Comment:
I had to ignore typin cause mypy was failing with `"str" has no attribute "value" ` error but debug_ddp is an enum any suggestions?

Reply:
The problem is in line 40 of `experiment_config.py`. You are marking the flag as `str`, but it should be `DebugDDPOptions`

Reply:
Yees, thanks :)